CONTRIBUTING.md

```diff
@@ -49,6 +49,35 @@ cargo run python install
 
 The storage directory can be configured with `UV_PYTHON_INSTALL_DIR`.
 
+### Snapshot testing
+
+uv uses [insta](https://insta.rs/) for snapshot testing. It's recommended (but not necessary) to use
+`cargo-insta` for a better snapshot review experience:
```

Comment:
I would personally install this through `cargo install cargo-insta`. I'm curious to hear from others on the team.

Reply:
Yeah I would as well